CMakeLists.txt

```diff
@@ -220,9 +217,9 @@ if ("${STATIC_BUILD}")
   endif ()
 else ()
   set (blupimania_DEPS
-    ${SDL2_LIBRARIES}
-    ${SDL2_MIXER_LIBRARIES}
-    ${SDL2_IMAGE_LIBRARIES}
+    SDL2::SDL2
+    SDL2_image::SDL2_image
+    SDL2_mixer::SDL2_mixer
     ${CURL_LIBRARIES}
```

Comment:
IIRC I had problems with this way on macOS or Windows/MSYS2.. But I must check again because I'm not sure about that.

Reply:
I tested against all operating systems and it works fine